webclient/dokument/templates/dokument/coor_doc.html

```diff
@@ -79,6 +79,10 @@
         global_map_can_edit = false
         addReadOnlyUniquePointToPoiLayer(corX, corY);
         {% endif %}
+        {% else  %}
+        if(!window.location.href.includes("model/zapsat")){
```

Comment:
@jiri-bartos-huld Bude to takto fungovat i pro `model/edit/<str:ident_cely>`? Tam také musí jít mapa upravovat...